### ISSUE TITLE
This repo is too large

### ISSUE THREAD

**Issue by lababidi** [h2oai/h2o-tutorials#113]
I was on hotel wifi and tried to clone this repo and found that it was nearing 800MB in size. I'd recommend changing the structure so that each tutorial is in its own repo. This way I'm not downloading all the git commits and changes along with all the extra files used.

**Comment by saldistefano** [comment]
Have you tried a shallow clone?

git clone --depth 1


Sal

________________________________

**Comment by lababidi** [comment]
Yeah, I actually have an found it to be also quite large. I'm not sure why.
